bats_ai/core/models/spectrogram.py

```diff
@@ -271,7 +271,7 @@ def compressed(self):
         buf.seek(0)
         img_base64 = base64.b64encode(buf.getvalue()).decode('utf-8')
 
-        return img_base64, starts_, stops_
+        return img_base64, starts_, stops_, self.duration
```

Comment:
`self.duration` is referring to the millisecond duration of the original spectrogram object, not the compressed view.

Reply:
I know, that's what I want.  I use the original pixel size and the duration to figure out the pixels/ms.  The compressed view doesn't squash anything, it just cuts out data, right?